src/lib/support/jsontlv/TlvToJson.cpp

```diff
@@ -0,0 +1,356 @@
+/*
+ *
+ *    Copyright (c) 2023 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#include "lib/support/CHIPMemString.h"
+#include "lib/support/ScopedBuffer.h"
+#include <json/json.h>
+#include <lib/core/DataModelTypes.h>
+#include <lib/support/Base64.h>
+#include <lib/support/SafeInt.h>
+#include <lib/support/jsontlv/ElementTypes.h>
+#include <lib/support/jsontlv/TlvToJson.h>
+
+namespace chip {
+
+namespace {
+
+constexpr uint32_t kTemporaryImplicitProfileId = 0x1122;
```

Comment:
I don't understand why this value is different from the one in the JsonToTlv conversion.  Please at least document why, or why it does not matter, or something.

Reply:
Added comment and ensured value is the same.